lcdblib/utils/chrom_convert.py

```diff
@@ -143,25 +148,47 @@ def convertFeature(f, mapper):
 
 def pybedtools_convert(input, output, mapper):
     """ Use pybedtools to convert chromosomes in BED, GTF, or GFF. """
-    pybedtools.BedTool(input).each(convertFeature, mapper).saveas(output)
+    if input == '-':
+        # Use STDIN
+        bt = pybedtools.BedTool(sys.stdin.read(), from_string=True)
```

Comment:
Never thought about having to do this. Just tested it though, you only need to provide sys.stdin, so:

```python
bt = pybedtools.BedTool(sys.stdin)
```